src/components/BulkMetadata/store/model.ts

```diff
@@ -0,0 +1,20 @@
+import { CustomFile } from '@amsterdam/bmi-component-library';
+
+export interface IBulkMetadataFile {
+	uploadedFile: CustomFile
+	metadata?: IBulkMetadataField;
+}
+
+export interface IBulkMetadataField {
+	id: string;
+	label: string;
+	value: string | number | boolean;
```

Comment:
kunnen we dit niet verder opsplitsen naar TextBulkMetadataField SelectBulkMetadataField, etc.

Reply:
Dat zou kunnen, welke velden gaat het dan om en hoe zien die eruit?